README.md

```diff
@@ -9,34 +9,58 @@ Facilitates participants trading accomodations and answering the critical questi
 * Django is good at managing relations and providing an api for controlling models. Used for the Room and Guest API model.
 * React is good at consuming stuffs and making things look good while they flossing.
 
+# Environment / Configuration
+
+Configuration is handled through environment variables, which are stored encrypted in GitHub. Secret management is handled through the `./scripts/secrets` script. You must have a file named `.secret` in the top level of the Git repository. Contact an adult for the contents of this file.
+
+* `./scripts/decrypt` generate the `secrets.env` file from encrypted source
+* `./scripts/encrypt` encrypt the `secrets.env` file
+* `./scripts/show` display all the env vars in a format suitable for `eval`
+* `./scripts show VAR` display the contents of the desired env var, stripped of quotes
+
 # Local Dev
 
 ## Requirements
 
 * `make` (a classic)
 * Docker configured in a way that networking and files work
-* python3.8
+* python 3.8
+* no not believin' in yo self
 
 ## Frontend
 
 ```
-$ make frontend_dev
-$ make frontend_build
+$ REACT_ROOMBAHT_API_ENDPOINT="http://localhost:8000/" make frontend_dev
 ```
 
 This should build a docker image, use it to generate the react static, and then start react in dev mode listening on port 3000.
 
 ## Backend
 
 ```
-$ export SECRET_KEY="narrative"
-$ export EMAIL_HOST_PASSWORD="words"
+$ export ROOMBAHT_DEV="true"
+$ export ROOMBAHT_SECRET_KEY="narrative"
+$ export ROOMBAHT_EMAIL_HOST_PASSWORD="words"
+$ export ROOMBAHT_EMAIL_HOST_USER="words"
+$ export ROOMBAHT_SEND_MAIL="true"
```

Comment:
Been talking with @Index01 and we think the right path is a separate secrets file for local dev. Will still have at least some "secrets" because of the email.